crowbar_framework/app/controllers/api/upgrade_controller.rb

```diff
@@ -62,6 +62,16 @@ def services
     end
   end
 
+  api :POST, "/api/upgrade/nodes", "Initiate the upgrade of all nodes"
+  api_version "2.0"
+  # This is gonna initiate the upgrade of all nodes.
+  # The method runs asynchronously, so there's a need to poll for the status and possible errors
+  def nodes
+    # FIXME: implement this as asynchronous call!
```

Comment:
I do not understand, you said before _its ok not to catch any errors here_ 
I do not have any there. But I'm missing asynchronous implementation, but that would be part of some future PR.